Source/WebCore/dom/ScriptElement.cpp

```diff
@@ -170,7 +176,21 @@ bool ScriptElement::prepareScript(const TextPosition& scriptStartPosition)
     if (wasParserInserted && !hasAsyncAttribute())
         m_forceAsync = true;
 
-    auto sourceText = scriptContent();
+    String sourceText;
+
+    if (element().scriptExecutionContext()->settingsValues().trustedTypesEnabled) {
```

Comment:
I think this would read nicer if we put `*element().scriptExecutionContext()` into a local variable.